lidro/create_virtual_point/vectors/run_create_virtual_points.py

```diff
@@ -60,8 +64,8 @@ def lauch_virtual_points_by_section(
         # Calculate the length of the river
```

Comment:
ligne 61 :     au lieu de `if not points.empty and not points["points_knn"].isnull().all():`, tu peux mettre juste `else` vu que ça correspond exactement au complémentaire de ton `if points.empty or points["points_knn"].isnull().all():` ligne 51